### PR TITLE
add install for libmagic

### PR DESCRIPTION
**Pull request by camallen**
ensure the client can use python libmagic - ideally these would be pushed to dockerhub as offical images for reuse via jenkins file on master changes / releases

without the changes
```
>>> from panoptes_client import Panoptes, Project
Broken libmagic installation detected. The python-magic module is installed but can't be imported. Please check that both python-magic and the libmagic shared library are installed correctly. Uploading media other than images may not work.
```

with changes
```
>>> from panoptes_client import Panoptes, Project
>>> quit()
```

### CODE REVIEW

File: Dockerfile.stable [hunk 1]
```diff
@@ -1,3 +1,7 @@
 FROM python:3-alpine
 
+RUN apk update && \
```
Comment (adammcmaster) [inline Dockerfile.stable]: It's a minor thing, but with `--no-cache` you don't need to call `apk update`.

File: Dockerfile.stable [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+    apk add --no-cache libmagic
+
+RUN pip install python-magic
```
Comment (adammcmaster) [inline Dockerfile.stable]: This shouldn't be needed because it's in the requirements list in `setup.py`.

Comment (adammcmaster) [reply Dockerfile.stable]: Just the `pip install` line I mean.

**Comment by PmasonFF** [comment]
Those attempting to install on a Windows 64 bit system require a different version of python-magic - python_magic-win64
Attempting the standard install from the requirements fails since that specifies python_magic

**Comment by adammcmaster** [comment]
Interesting. Could you open a new issue for that, @PmasonFF? We should certainly make this easier to use on Windows.

**Pull request merged**